test/integration/containerized/test_cli_containerized.py

```diff
@@ -29,3 +33,36 @@ def test_playbook_run(cli, skip_if_no_podman):
 def test_provide_env_var(cli, skip_if_no_podman, test_data_dir):
     r = cli(['run', os.path.join(test_data_dir, 'job_env'), '-p', 'printenv.yml'])
     assert 'gifmyvqok2' in r.stdout, r.stdout
+
+
+def test_cli_kill_cleanup(cli, test_data_dir, container_runtime_installed):
```

Comment:
The `container_runtime_installed` fixture uses the first found run time. This test should be run against all supported runtimes. We have bigger issues in that our CI is missing `podman`, but tests should be written to run against all available runtimes, not just the first found. I have a lot of existing tests to fix, but new ones should do the right thing (unless we are testing a `docker` or `podman` only feature).

```suggestion
@pytest.mark.parametrize('runtime', ('podman', 'docker'))
def test_cli_kill_cleanup(cli, test_data_dir, runtime):
```

Reply:
(done) ✅